### ISSUE TITLE
Opening Atom in a directory with a local git config applies it to the global git config

### ISSUE THREAD

**Issue by petebachant** [atom/atom#21708]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Steps to Reproduce

1. Create an empty git repo and inside run `git config user.name "Test Name"`
2. Run `atom .` inside that git repo directory
3. Run `git config --global --list` to see now that the global user name is "Test Name"

**Reproduces how often:**

Every time.

### Versions

```
Atom    : 1.53.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0
```

```
apm  2.5.2
npm  6.14.8
node 12.4.0 x64
atom 1.53.0
python 3.7.9
git 2.29.2.windows.2
visual studio 2015
```

Running on Windows 10 Pro 64 bit.

**Comment by mkismy** [comment]
Probably a duplicate of atom/github#2557, atom/github#2558 which should be fixed soon.

**Comment by smashwilson** [comment]
Yeah, this is a duplicate of atom/github#2622 and atom/github#2557 and probably a few others that I can't find right now. Already fixed by atom/github#2587. The latest package version lets you choose explicitly:

![image](https://user-images.githubusercontent.com/17565/107086387-ee5a9a00-67c7-11eb-9199-07aaed60a72b.png)

Try switching to [Atom Nightly](https://atom.io/nightly) if you want this behavior sooner rather than later!
